radio/src/rtc.h

```diff
@@ -55,6 +55,14 @@ void rtcSetTime(const struct gtm * tm);
 gtime_t gmktime (struct gtm *tm);
 uint8_t rtcAdjust(uint16_t year, uint8_t mon, uint8_t day, uint8_t hour, uint8_t min, uint8_t sec);
 
+
+#define RTCBKBR_BASE  ((void *)(&(RTC->BKP0R))) // base address of RTC backup registers
+#define RTCBKP0R 0                              // RTC backup register 0
+#define RTCBKP1R 1                              // RTC backup register 1
+
+uint32_t getRTCBKPR(uint8_t RTCBKPRegister);
+void setRTCBKPR(uint8_t RTCBKPRegister, uint32_t value);
```

Comment:
No need to define these, as they are already defined in HAL/LL:
```c++
#define LL_RTC_BKP_DR0                     0x00000000U
#define LL_RTC_BKP_DR1                     0x00000001U

void LL_RTC_BAK_SetRegister(RTC_TypeDef *RTCx, uint32_t BackupRegister, uint32_t Data);
uint32_t LL_RTC_BAK_GetRegister(RTC_TypeDef *RTCx, uint32_t BackupRegister);
```

Reply:
done - please review changes thoroughly